peterjohnson/PETER_JOHNSON.md

```diff
@@ -0,0 +1,77 @@
+with something that you are curious about and find engaging.
+
+## Develop skills that can help you solve problems
+
+Simply put, you need to develop some skills if you want to work for someone, whether that be for your employer or for 
+one of your customers. Find out what skills you bring to the table, start developing skills as soon as you can, don’t 
+fear perfection and just start and take small steps each day or week. One of the first skills I wanted to learn in 2011 
+was the ability to code in Java, and I started by finding opportunities to apply what I was learning at work and set 
+myself goals for studying so much each week. Talk to you manager, a colleague or a friend and see how people can support 
+you, ideally you want to find someone who can hold you accountable. You need to be vocal and direct about what you want 
+to learn, don’t feel it’s a weakness to discuss what you are learning with others. I’ve learned a lot from having 
+discussions with colleagues and peers by testing my ideas and understanding. Pick one skill, focus and practice it every 
+day or week. 
+Malcolm Gladwell states in his Outliers book it takes 10,000 hours of practice to master a skill, so start now!
+
+## Set goals and work towards them every week
+
+Develop a growth mindset, this is very important. Working in technology requires continuous learning, it’s one of the 
+reasons I like working in the industry as I get the opportunity to learn new technologies, approaches or methodologies 
+as part of my daily work. You must set yourself some goals and be accountable to working towards them, I write my goals 
+in a moleskin notebook that I keep by my bedside so that I can review them easily. Once you have set your goal, break 
+the goal down to actionable tasks that you can schedule in your weekly calendar to work on. For example, if my goal was 
+to learn to drive, then I would schedule a driving lesson or two a week, this is no different from wanting to schedule 
+time for studying algorithms each week if my goal was to become a better software engineer. Never give up on yourself!
+
+## Learn from failure fast and continue to grow
+
+Nobody likes failure, especially when you describe an event in such a way, whether that be a failed project, business or 
+promotion. I tend to find I have learnt more from failure than I have from success, I see failure as a learning 
+opportunity so prefer to call it that.
+Whether you want to be a SDET, manager or software engineer find out what skills you lack or are weak in and create a 
+plan to work towards improving them. Ask for feedback on your work and skills and be prepared to actively listen to 
+other opinions, both positive and constructive feedback. Even when something seemingly went well, ask for feedback! 
+You can then form a plan on how to improve, for example I remember giving a presentation to an engineering team on a 
```

Comment:
Another area a semicolon could be used rather than a comma:

Current: `You can then form a plan on how to improve, for example I remember giving a presentation to an engineering team on a `

Recommended: `You can then form a plan on how to improve; for example I remember giving a presentation to an engineering team on a `